src/pal/src/configure.cmake

```diff
@@ -73,8 +73,33 @@ check_function_exists(directio HAVE_DIRECTIO)
 check_function_exists(semget HAS_SYSV_SEMAPHORES)
 check_function_exists(pthread_mutex_init HAS_PTHREAD_MUTEXES)
 check_function_exists(ttrace HAVE_TTRACE)
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+set(CMAKE_REQUIRED_LIBRARIES unwind unwind-generic)
```

Comment:
I'm not sure this would quite achieve what we want. Whilst we normally use the standard libunwind on Linux LLVM do have a libunwind also. I'm not sure if it's worth adding here but maybe it should be considered that there may be either `libunwind`, `libunwind-generic` or both. If it was possible to account for this then these two tests could be used everywhere. https://github.com/dotnet/coreclr/blob/master/src/pal/src/CMakeLists.txt#L211 shows the various unwind library names.